### PR TITLE
Add to emitResponse utf-8 support #1199

### PR DESCRIPTION
**Pull request by Kagayakashi**
In json emitResponse added utf8 support for issue #1199

### CODE REVIEW

**Comment by spawnia** [comment]
Please add a failing test case first.

**Comment by Kagayakashi** [comment]
> Please add a failing test case first.

Helper -> emitResponse method dont have return value, there is direct "echo".
So I added test semi hardcoded.

Will be fine?

**Comment by spawnia** [comment]
https://phpunit.readthedocs.io/en/9.5/writing-tests-for-phpunit.html#testing-output

File: tests/Server/HelperTest.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+     */
+    public function testSendResponseWithUtf8Support(): void
+    {
+        $dataArray = [
+            'data' => [
+                'user' => 'Петя'
+            ]
+        ];
+        
+        $expected = json_encode($dataArray, JSON_UNESCAPED_UNICODE);
```
Comment (spawnia) [inline tests/Server/HelperTest.php:21]: That test is repeating the implementation, it should rather spell out the correct expected result.

File: tests/Server/HelperTest.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+        $helper = new Helper();
+        $helper->sendResponse($dataArray);
+
+        $this->expectOutputString($expected);
```
Comment (spawnia) [inline tests/Server/HelperTest.php:26]:
```suggestion
 
        $this->expectOutputString($expected);
        $helper->sendResponse($dataArray);
```

File: tests/Server/HelperTest.php (hunk 1 above)
Comment (Kagayakashi) [reply tests/Server/HelperTest.php:21]: I just also added "JSON_UNESCAPED_UNICODE" to actual code along with test.
Current master version dont have this.
Should it be replaced with strictly written json?

Like {"data":{"user":"Петя"}}

**Pull request merged**